pkg/crc/machine/start.go

```diff
@@ -923,47 +920,14 @@ func logBundleDate(crcBundleMetadata *bundle.CrcBundleInfo) {
 	}
 }
 
-func ensureRoutesControllerIsRunning(sshRunner *crcssh.Runner, ocConfig oc.Config, preset crcPreset.Preset, bundleInfo *bundle.CrcBundleInfo) error {
-	bin, err := json.Marshal(v1.Pod{
-		TypeMeta: metav1.TypeMeta{
-			Kind:       "Pod",
-			APIVersion: "v1",
-		},
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      "routes-controller",
-			Namespace: "openshift-ingress",
-		},
-		Spec: v1.PodSpec{
-			ServiceAccountName: "router",
-			Containers: []v1.Container{
-				{
-					Name:            "routes-controller",
-					Image:           getRouterControllerImage(preset, bundleInfo),
-					ImagePullPolicy: v1.PullIfNotPresent,
-				},
-			},
-		},
-	})
-	if err != nil {
-		return err
-	}
-	if err := sshRunner.CopyData(bin, "/tmp/routes-controller.json", 0644); err != nil {
-		return err
-	}
-	_, _, err = ocConfig.RunOcCommand("apply", "-f", "/tmp/routes-controller.json")
+func ensureRoutesControllerIsRunning(ocConfig oc.Config) error {
+	_, _, err := ocConfig.RunOcCommand("apply", "-f", "/opt/route_controller.yaml")
```

Comment:
Can this go to `/usr/share/crc/routes_controller.yaml` or such?

Reply:
@cfergeau any specific reason because we are storing the `kubeconfig` in the same location so didn't want to change it.

Reply:
I know there are other files there, I made a comment at the time, but I should have pushed stronger :) 
/opt is an odd place to put crc-specific files, especially without subdirectories.

Reply:
@cfergeau updated.

Reply:
https://github.com/crc-org/snc/pull/738#issuecomment-1591023529

Reply:
`/opt/crc` makes perfectly sense to me.

Reply:
`/usr/share` is RO in RHCOS, but `/usr/share/local/crc/` is also a 'good' location, not owned by packages.